### ISSUE TITLE
add an hover effects on profile items

### ISSUE THREAD

**Issue by aayusranjan** [codingbrunches/codingBrunch-community#5]

**Comment by ParthasarathiMahana** [comment]
hi @aayusranjan can you explain more on the issue I will love to work on this.

**Comment by aayusranjan** [comment]
Yes! you need to add an hover effect when someone take their cursor on any profile(card), maybe you can do it normal at first time then after hovering, generate box shadows
![Screenshot 2022-10-07 060806](https://user-images.githubusercontent.com/63737303/194762115-a049fc87-474e-40d3-87c4-e0c994bd2044.png)

**Comment by aayusranjan** [comment]
![Screenshot 2022-10-09 231118](https://user-images.githubusercontent.com/63737303/194771654-ef8c2ae6-1cd5-4337-a681-4ed671d3664b.png)
Here is the reference you can use, the third card is after hovering
